sdk/eventhub/Azure.Messaging.EventHubs/tests/Infrastructure/TestCategory.cs

```diff
@@ -17,5 +17,8 @@ public static class TestCategory
 
         /// <summary>The associated test is meant to verify a scenario from end-to-end which is safe to run in isolation; it has no external dependencies</summary>
         public const string EndToEnd = "EndToEnd";
+
+        /// <summary>The associated test should not be included when Visual Studio is performing "Live Unit Testing" runs.</summary>
+        public const string DisallowVisualStudioLiveUnitTesting = "SkipWhenLiveUnitTesting";
```

Comment:
This doesn't seem to be used.

Reply:
It will be.  There are no live tests at present.

Reply:
I tend to avoid adding APIs that are not in use.

Reply:
I'm building out a series of changes that will make use of it and attempting to push out PRs in units that make sense for both review granularity and to keep others unblocked.  There's will be dangling items until the work stream is complete.